chapters/hpmor-chapter-023.tex

```diff
@@ -40,7 +40,7 @@ \chapter{Belief in Belief}
 
 “The secret of blood,” said Harry Potter, an intense look on his face, “is something called deoxyribonucleic acid. You don’t say that name in front of anyone who’s not a scientist. Deoxyribonucleic acid is the recipe that tells your body how to grow, two legs, two arms, short or tall, whether you have brown eyes or green. It’s a material thing, you can \emph{see} it if you have microscopes, which are like telescopes only they look at things that are very small instead of very far away. And that recipe has two copies of everything, always, in case one copy is broken. Imagine two long rows of pieces of paper. At each place in the row, there are two pieces of paper, and when you have children, your body selects one piece of paper at random from each place in the row, and the mother’s body will do the same, and so the child also gets two pieces of paper at each place in the row. Two copies of everything, one from your mother, one from your father, and when you have children they get one piece of paper from you at random in each place.”
 
-As Harry spoke, his fingers ranged over the paired scraps of paper, pointing to one part of the pair when he said “from your mother”, the other when he said “from your father”. And as Harry talked about picking a piece of paper at random, his hand pulled a Knut out of his robes and flipped it; Harry looked at the coin, and then pointed to the top piece of paper. All without a pause in the speech.
+As Harry spoke, his fingers ranged over the paired scraps of paper, pointing to one part of the pair when he said “from your mother,” the other when he said “from your father”. And as Harry talked about picking a piece of paper at random, his hand pulled a Knut out of his robes and flipped it; Harry looked at the coin, and then pointed to the top piece of paper. All without a pause in the speech.
```

Comment:
I think this is a quotation too: the author is quoting Harry rather than reporting his speech when it occurs.